webapi/src/main/scala/org/knora/webapi/responders/v2/ontology/OntologyLegacyRepo.scala

```diff
@@ -0,0 +1,27 @@
+package org.knora.webapi.responders.v2.ontology
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import org.knora.webapi.InternalSchema
+import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.v2.responder.ontologymessages.ReadClassInfoV2
+import org.knora.webapi.messages.v2.responder.ontologymessages.ReadOntologyV2
+
+object OntologyLegacyRepo {
```

Comment:
Why do you call it "legacy"? What is the follow-up plan for it?

Reply:
It is `Legacy` because it is returning `Future`s. The plan is that we will have only zio implementations, which should be easier to use once this abstraction is used on more places. 

I tried to align the method signatures with the `Repository` trait which I cannot use here but this should simplify migration to zio later.

Reply:
I'm still wondering why you use "legacy" here and not in other places where Futures are returned. But I see your intention, so it's OK for me. It's `Z` for `ZIO` stuff, `legacy` for `Future` stuff.